src/Azure.IIoT.OpcUa/tests/Encoders/Encoders/JsonEncoderDecoderTests.cs

```diff
@@ -46,7 +46,7 @@ public void ReadWriteProgramDiagnostic2DataTypeStream()
                     new Argument("foo2",
                         new NodeId(33), -1, "fdsadfsdaf") { ArrayDimensions = Array.Empty<uint>() },
                     new Argument("adfsdafsdsdsafdsfa",
-                        new NodeId("absc"), 1, "fsadf  sadfsdfsadfsd") { ArrayDimensions = Array.Empty<uint>() },
+                        new NodeId("absc", 0), 1, "fsadf  sadfsdfsadfsd") { ArrayDimensions = Array.Empty<uint>() },
```

Comment:
>new NodeId("absc", 0) [](http://example.com/codeflow?start=24&length=21)

fun facts: try new NodeId("s=absc") or "s=absc" or new NodeId("ns=0;absc") should all create the same NodeId